martin/src/cog/mod.rs

```diff
@@ -306,18 +324,9 @@ fn get_meta(path: &PathBuf) -> Result<Meta, FileError> {
         zoom_and_tile_across_down.insert(zoom, (tiles_across, tiles_down));
     }
 
-    let min_zoom = zoom_and_ifd
-        .keys()
-        .min()
-        .ok_or_else(|| CogError::NoImagesFound(path.clone()))?;
-
-    let max_zoom = zoom_and_ifd
-        .keys()
-        .max()
-        .ok_or_else(|| CogError::NoImagesFound(path.clone()))?;
     Ok(Meta {
-        min_zoom: *min_zoom,
-        max_zoom: *max_zoom,
+        min_zoom: 0,
```

Comment:
Just that this is intentional:
COGs always include data for the zoom level `0`?

Reply:
Add a is_empty() check now.

Reply:
`is_empty` does not seem relevant to me.

Lets rephrase the question:
Do COGs always include data for the zoom level `0`?

Reply:
A valid COG must have at least one image